stations.csv

```diff
@@ -27412,3 +27412,93 @@ id;name;slug;uic;uic8_sncf;latitude;longitude;parent_station_id;country;time_zon
 29559;Vatra Dornei;vatra-dornei;;;47.3447576;25.3521039;;RO;Europe/Bucarest;t;f;f;t;f;f;;;f;;f;;f;;f;;f;15778;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29560;Zalău;zalau;;;47.1854562;23.0573324;;RO;Europe/Bucarest;t;f;f;t;f;f;;;f;;f;;f;;f;;f;15788;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
 29561;Țăndărei;tandarei;;;44.6488718;27.6438672;;RO;Europe/Bucarest;t;f;f;t;f;f;;;f;;f;;f;;f;;f;15698;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
+29562;Bale;bale;;;45.040703;13.783752;;HR;Europe/Zagreb;t;f;f;t;f;f;;;f;;f;;f;;f;;f;9618;t;;f;;f;;f;;;;f;;f;;f;;f;;f;f;;;;;;;;;;;;;;;;;;;
```

Comment:
Will it be clear that it it not Basel, Switzerland in French?

Reply:
Updated to add the country_hint